src/sentry/api/endpoints/group_activities.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases import GroupEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import Activity
+
+
+class GroupActivitiesEndpoint(GroupEndpoint, EnvironmentMixin):
+    def get(self, request, group):
+        """
+        Retrieve all the Activities for a Group
+        """
+        activity = Activity.get_activities_for_group(group, num=100)
+        return Response(
+            {
+                "activity": serialize(activity, request.user),
+            }
+        )
```

Comment:
We should use `self.paginate` here so that anyone consuming this endpoint can get additional pages as necessary. Can also drop the `num=100` since our default `page_page` is 100.